map/framework.hpp

```diff
@@ -394,7 +394,7 @@ class Framework : public search::ViewportSearchCallback::Delegate,
   /// Guarantees that listener is called in the main thread context.
   void SetCurrentCountryChangedListener(TCurrentCountryChanged const & listener);
 
-  vector<MwmSet::MwmId> GetMwmsByRect(m2::RectD const & rect) const;
+  vector<MwmSet::MwmId> GetMwmsByRect(m2::RectD const & rect, bool coarse) const;
```

Comment:
Are you sure that "coarse" fits well here? From my point of view, "rough" is more common in similar cases.